driver: wifi: nxp: 11k neighbor request support for embedded supplicant

gangli02 opened the pull request:
Add 11k neighbor request support for embedded supplicant.

gangli02 committed:
driver: wifi: nxp: 11k neighbor request support for embedded supplicant

Add 11k neighbor request support for embedded supplicant.

Signed-off-by: Gang Li <gang.li_1@nxp.com>

jukkar reviewed (commented):
Please split the commit to two pieces, one changing the NXP driver and one changing the wifi shell.

gangli02 commented:
> Please split the commit to two pieces, one changing the NXP driver and one changing the wifi shell.

Okay, let me update

gangli02 committed:
net: wifi: shell: fix 11k neighbor request cannot specify ssid

Fix the issue of sending neighbor report request failing
when specifying ssid.

Signed-off-by: Gang Li <gang.li_1@nxp.com>
